Tickets/dm 8636

mssgill opened the pull request:
Changes just to the Task template, and not the others, for now.

Commit:
Make Task template consistent with example tasks

This is incorporating our experience with the sfp tasks and bringing
it into the Task template.

Commit:
Add needed files to standalone template dir build

All the ones we were using in the sfp subdir, with small modifications
as needed.

jonathansick commented on templates/template.forTasks.rst:
```diff
@@ -5,96 +5,111 @@ Based on: https://dmtn-030.lsst.io/v/DM-7096/index.html#task-topic-type
+  Here we simply state in which module the task is implemented.
 
 See also
 =========
```
You can leave out the header "See also" or else devs might get confused about whether they should have a header here in addition to the `.. seealso::` directive.

mssgill replied:
Ok, cool, done

jonathansick commented on templates/template.forTasks.rst:
```diff
@@ -5,96 +5,111 @@ Based on: https://dmtn-030.lsst.io/v/DM-7096/index.html#task-topic-type
 Template for Tasks
 ##################
```
Might want to style this after the title we intend. Like `ExampleTask`.

mssgill replied:
Ok, done

jonathansick commented on templates/index.rst:
```diff
@@ -0,0 +1,29 @@
+.. toctree::
+   :maxdepth: 1
+
+   template.forTasks
```
If we make the title of the task template realistic, you can give it a sensible title here as:

```
Task Template <template.forTasks>
```

mssgill replied:
Cool, did this, also

Commit:
Remove See Also header

Commit:
Rename template.forTasks to ExampleTask

At JSick's suggestion, makes it better in more parallel.

Commit:
Modify ExampleTask entry in index.rst

To be consistent with name change inside Task Template file.

Commit:
Modify ExampleTask to Task Template in index.rst

Wanted this to be the name instead.

Commit:
Remove the 'Entrypoint' section

We found this is not being useful for us, everything in the `run`
interface is now incorporated into the API Usage page which is linked
to from the SeeAlso box above.

jonathansick commented:
Looks like this work has moved to DM-8723.
